ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecs.java

```diff
@@ -647,12 +647,15 @@ static ProtocolSpecBuilder shanghaiDefinition(
       final boolean quorumCompatibilityMode,
       final EvmConfiguration evmConfiguration) {
 
+    // extra vaiables need to support flipping the warm coinbase flag.
```

Comment:
useful comments 👍

Reply:
I wrote that just a month ago, and If I had to look 3 files deep to see why it was there... maybe just maybe I should comment it.